arwenmandos/stepCheckAccounts.go

```diff
@@ -274,8 +274,49 @@ func checkTokenInstances(
 					expectedInstance.Balance.Original,
 					accountInstance.Value))
 			}
+			if !bytes.Equal(expectedInstance.Creator.Value, accountInstance.TokenMetaData.Creator) {
```

Comment:
Use:
`expectedInstance.Creator.Check(accountInstance.TokenMetaData.Creator)`
To also allow for the check to be ignored, when the developer writes `"*"` .
The same goes for all the other checks below.

Reply:
Done